apps/test/unit/templates/studioHomepages/CoteacherInviteNotificationTest.jsx

```diff
@@ -17,16 +18,36 @@ describe('CoteacherInviteNotification', () => {
       status: 'invited',
       instructor_name: 'Linus',
     },
+    coteacherInviteForPl: {
+      id: 2,
+      invited_by_name: 'Ms. Frizzle',
+      invited_by_email: 'magicSchoolBus@code.org',
+      section_name: 'Section for teachers',
+      status: 'invited',
+      instructor_name: 'Larry',
+    },
   };
 
-  it('renders nothing if there is no coteacher invite', () => {
+  it('renders no notification for classroom sections if there is no coteacher invite', () => {
     DCDO.set('show-coteacher-ui', true);
     const wrapper = shallow(
       <CoteacherInviteNotification {...defaultProps} coteacherInvite={null} />
     );
     expect(wrapper.find(Notification).length).to.equal(0);
   });
 
+  it('renders no notification for PL sections if there is no coteacher invite', () => {
```

Comment:
One more test case: render nothing if `isForPl = true`, no pl coteacher invite but there is a regular coteacher invite.

Reply:
I think I might be missing something here, I think this is what this test already does...